Mathlib/Analysis/InnerProductSpace/Positive.lean

```diff
@@ -55,12 +165,35 @@ def IsPositive (T : E →L[𝕜] E) : Prop :=
 theorem IsPositive.isSelfAdjoint {T : E →L[𝕜] E} (hT : IsPositive T) : IsSelfAdjoint T :=
   hT.1
 
-theorem IsPositive.inner_nonneg_left {T : E →L[𝕜] E} (hT : IsPositive T) (x : E) :
+theorem IsPositive.re_inner_nonneg_left {T : E →L[𝕜] E} (hT : IsPositive T) (x : E) :
     0 ≤ re ⟪T x, x⟫ :=
   hT.2 x
 
+theorem IsPositive.re_inner_nonneg_right {T : E →L[𝕜] E} (hT : IsPositive T) (x : E) :
+    0 ≤ re ⟪x, T x⟫ := by rw [inner_re_symm]; exact hT.re_inner_nonneg_left x
+
+open ComplexOrder in
+theorem isPositive_iff_isSelfAdjoint_and_inner_nonneg_left (T : E →L[𝕜] E) :
+    IsPositive T ↔ IsSelfAdjoint T ∧ ∀ x, 0 ≤ ⟪T x, x⟫ := by
+  unfold IsPositive
+  rw [and_congr_right_iff]
+  intro hT
+  apply forall_congr'
+  intro x
+  rw [nonneg_iff (z := ⟪T x, x⟫), ← conj_eq_iff_im, inner_conj_symm]
+  nth_rw 2 3 [show T x = (T : E →ₗ[𝕜] E) x by rfl]
+  rw [isSelfAdjoint_iff_isSymmetric.mp hT]
+  unfold reApplyInnerSelf
+  simp
+
+open ComplexOrder in
+theorem IsPositive.inner_nonneg_left {T : E →L[𝕜] E} (hT : IsPositive T) (x : E) :
+    0 ≤ ⟪T x, x⟫ :=
+  ((isPositive_iff_isSelfAdjoint_and_inner_nonneg_left T).mp hT).right x
+
+open ComplexOrder in
 theorem IsPositive.inner_nonneg_right {T : E →L[𝕜] E} (hT : IsPositive T) (x : E) :
```

Comment:
Can you make this match the "left" version above? i.e. define it in terms of the complex order.